csrc/quantization/gptq_marlin/gptq_marlin.cu

```diff
@@ -832,6 +863,7 @@ __global__ void Marlin(
   FragS act_frag_s[2][4][4];             // For act-order
   int frag_qzp[2][num_ints_per_thread];  // Zero-points
   FragZP frag_zp;                        // Zero-points in fp16
+  FragZPF frag_zpf[2][4];                // float16 zero-points
```

Comment:
nit: couldn't this just be `FragZP frag_zpf[2];` or something like that? since `FragZPF` is `Vec<half2, 1>`  and `FragZP` is `Vec<half2, 4>` (same for bfloat) (i.e. we can just make `FragZPF` a vec of 4 which makes it equivalent to  `FragZP` instead of `[4]` or am I missing something)

Not critical

Reply:
Thanks! You're right, the new type was not needed and this simplifies the code and templated functions a bit.